### PR TITLE
MVS-692 Add "TEST WEBSITE - NOT LIVE" to Registration and POD applications

### PR DESCRIPTION
**Pull request by dmcook5208**
## :floppy_disk: [Code Check]
_Make sure you've checked/done the following before making this PR!_
* Does your code build/run?
* Does your design approach make sense?
* Does your work achieve the intention of the card?
* Does your code have any unintentional side effects -- have you tested a full workflow & checked the logic branches you touched?
* Did you make a unit test for your code (as appropriate)?
* Is your branch relatively up to date with the Development branch (within ~2 commits)
* Have you moved the card to "Needs QA" in Jira

## :flipper: [JIRA Sprint Card]

https://mass-immunization-system.atlassian.net/browse/MVS-692

## :newspaper: [Work Description]

Added "TEST WEBSITE - NOT LIVE" (with alert icon) to the top of the Registration and POD applications (changes made to the App.vue file of each application)

## :vertical_traffic_light: [Testing/QA Notes]

Build and load the Registration app (using any method).  Verify that there is a red bar along the top of the website that says "(alert icon) TEST WEBSITE - NOT LIVE"
Build and load the POD app (using any method).  Verify that there is a red bar along the top of the website that says "(alert icon) TEST WEBSITE - NOT LIVE"

### CODE REVIEW

**Comment by thespaniard514** [comment]
Not 100% necessary, but to keep the App.vue file clean, this could go into its own "partial" similar to the Header.vue partial. See `page/application/partials/Header.vue` 

Later down the road we can figure out a way to share these components/partials between Pat and POD apps using npm so we can cut down on duplication

File: PatientRegistrationApp/src/App.vue [hunk 1]
```diff
@@ -1,6 +1,14 @@
 <template>
   <v-app id="Patient-registration">
     <v-main>
+      <v-system-bar
```
Comment (thespaniard514) [inline PatientRegistrationApp/src/App.vue:4]: v-if with a boolean attached to our config (could read from our .env file for configuration by users of system)

File: PointOfDispensingApp/src/App.vue [hunk 2]
```diff
@@ -1,5 +1,14 @@
 <template>
   <v-app>
+    <v-system-bar
```
Comment (thespaniard514) [inline PointOfDispensingApp/src/App.vue:3]: same as above, v-if

**Review by jensen0914 — COMMENTED** [review]
Ideally, whether to display this warning should be a configuration read from a file on the server as eventually you will want to NOT display it. 

You could add a properties.json/.env like file that is part of the codebase that is put on the server. It would be read when you Vue app is initialized (and could also eventually contain other configurations that are site specific).

**Review by thespaniard514 — APPROVED** [review]
Looks good!

**Review by jensen0914 — APPROVED** [review]

**Pull request merged**